annotation-utils/src/main/java/edu/isi/vista/annotationutils/ExtractAnnotationStats.kt

```diff
@@ -290,7 +526,18 @@ class ExtractAnnotationStats {
     }
 }
 
-data class DocumentAnnotation(val documentName: String, val user: String, val eventType: String)
-data class EventSpan(val dependent: Int, val governor: Int, val relation_type: String)
-data class AnnotationStats(val total: Int, val byUser: Map<String, Int?>, val byEventType: Map<String, Int?>)
+data class SentenceAnnotation(
+        val sentence_id: String,
+        val user: String,
+        val eventType: String,
+        val corpus: String,
+        val negativeExample: Boolean
+)
+data class AnnotationStats(
+        val total: Int,
+        val byUser: Map<String, Int?>,
+        val byEventType: Map<String, Int?>,
+        val byCorpusPositive: Map<String, Int?>?,
+        val byCorpusNegative: Map<String, Int?>?
```

Comment:
Why are these all `Int?` and two of the maps `?` themselves? Can you make it so that if the values are going to be null, there just wouldn't be a key? And the mappings should never be null, they should be present but empty.